plugin/externalPluginEntry.go

```diff
@@ -530,8 +558,8 @@ func (e *externalPluginEntry) unmarshalSchemaGraph(stdout []byte) (*linkedhashma
 	if len(rawGraph) <= 0 {
 		return nil, fmt.Errorf("expected a non-empty JSON object")
 	}
-	if rawGraph[e.typeID] == nil {
-		return nil, fmt.Errorf("%v's schema is missing", e.typeID)
+	if rawGraph[rawTypeID] == nil {
+		return nil, fmt.Errorf("%v's schema is missing", e.rawTypeID())
```

Comment:
We have rawTypeID, use it directly instead of calling the function?

Reply:
Oh yeah, that was from an earlier iteration.